### PR TITLE
Introduce a mock score step for experimenting with predictors

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: thoth/adviser/steps/mock_score.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    def post_run(self) -> None:
+        """Print the generated history after the run."""
+        packages = {}
+        for key, value in self._score_history.items():
+            packages.setdefault(key[0], []).append((key, value))
+
+        for key, value in packages.items():
+            packages[key] = sorted(value, key=operator.itemgetter(1), reverse=True)
+
+        print("-"*10, " Mock score report ", "-"*10, file=sys.stderr)
```
Comment (harshad16) [inline thoth/adviser/steps/mock_score.py:72]: use print instead of logging info, is this intentional?

File: thoth/adviser/steps/__init__.py [hunk 2]
```diff
@@ -28,5 +29,6 @@
 __all__ = [
     "AICoEReleasesStep",
     "CvePenalizationStep",
+    # "MockScoreStep",
```
Comment (harshad16) [inline thoth/adviser/steps/__init__.py:32]: was this commented out, intentionally or missed ?

File: thoth/adviser/steps/mock_score.py (hunk 1 above)
Comment (fridex) [reply thoth/adviser/steps/mock_score.py:72]: Yes, this step is used solely for debugging to experiment with adviser locally.

Comment (harshad16) [reply thoth/adviser/steps/mock_score.py:72]: ok

**Comment by harshad16** [comment]
@fridex ,can you resolve the conflicts
this looks good to me

**Pull request merged**